src/renderer/api/catalog-entity-registry.ts

```diff
@@ -0,0 +1,61 @@
+import { action, observable } from "mobx";
+import { broadcastMessage, subscribeToBroadcast } from "../../common/ipc";
+import { CatalogCategory, CatalogEntity, CatalogEntityData } from "../../common/catalog-entity";
+import { catalogCategoryRegistry, CatalogCategoryRegistry } from "../../common/catalog-category-registry";
+import "../../common/catalog-entities";
+
+export class CatalogEntityRegistry {
+  @observable protected _items: CatalogEntity[] = observable.array([], { deep: true });
+
+  constructor(private categoryRegistry: CatalogCategoryRegistry) {}
```

Comment:
Why is the entity registry getting passed the category registry. I figured they would be singletons.